actions/molecule-test/action.yml

```diff
@@ -11,6 +11,10 @@ inputs:
     description: Path to use when checking out the repository
     required: false
     default: ./
+  tests_path:
```

Comment:
Generally in GitHub Actions, peeps tend to use `tests-path` rather than `tests_path`